tests/test_conflicts.py

```diff
@@ -71,23 +72,23 @@ def test_summarise_conflicts(data_archive, cli_runner):
             "",
         ]
 
-        r = cli_runner.invoke(["conflicts", "-s", "-o", "json"])
+        r = cli_runner.invoke(["conflicts", "-s", "-o", "json", "--output=-"])
```

Comment:
Keep the default behaviour for output working - if you don't specify output, it should default to stdout.